links/gql_websocket_link/lib/src/link.dart

```diff
@@ -120,7 +120,8 @@ class WebSocketLink extends Link {
     this.graphQLSocketMessageDecoder = _defaultGraphQLSocketMessageDecoder,
     this.initialPayload,
     this.inactivityTimeout,
-  }) : assert(uri == null || (channelGenerator == null)) {
```

Comment:
Please remove the extra parentheses.
aassert(uri == null || channelGenerator == null)`

Reply:
What do you mean? This line is deleted/modified in this PR and does not include extra parentheses around channelGenerator anyways.

Reply:
My mistake, I was confused by the change you made.